tp2/Pacmania/Assets/Scripts/GameLoader.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class GameLoader : MonoBehaviour 
+{
+	public GameObject gameManager;			//GameManager prefab to instantiate.
+	//public GameObject soundManager;			//SoundManager prefab to instantiate.
+
+
+	void Awake ()
+	{
+		//Check if a GameManager has already been assigned to static variable GameManager.instance or if it's still null
+		if (GameManager.instance == null)
+			//Instantiate gameManager prefab
```

Comment:
As the rest of the code has `{ ... }` in the if blocks, we could add here o remove the others. 
Personally I prefer the first option.